ddtrace/internal/ci_visibility/api/ci_base.py

```diff
@@ -95,7 +98,11 @@ def __init__(
         self._tags = initial_tags if initial_tags else {}
         self._children = None
 
+        # ITR-related attributes
         self._is_itr_skipped: bool = False
+        self._itr_skipped_count: int = 0
```

Comment:
Just to make sure I understand this correctly, this count is used to track the number times an item and its child items (tests and suites) are skipped?

Reply:
Yep. It's mostly used at the session and module level, so I could technically not count it at the suite or test level, but I'm not sure it's worth the `if` statement for "if it's a suite and we're in suite-skipping mode, don't count it" (especially since suite-skipping mode isn't a released feature).

Reply:
Where does this get used? Is this for some kind of dashboard to track this? 👀

Reply:
It's a tag on the spans, so you could do a search like this:
![image](https://github.com/DataDog/dd-trace-py/assets/136473744/7e613488-2e84-4e67-9293-79c1fa234430)
and get results like 
![image](https://github.com/DataDog/dd-trace-py/assets/136473744/d8af2794-78c2-4fbb-beda-9ffd1679214f)